components/SearchResultDetails.vue

```diff
@@ -10,6 +10,9 @@
             <div class="result-header mt-7 ml-4">
                 <span class="w-4 text-3xl font-bold pl-2 self-center">{{
                     healthcareProfessionalName
+                }}, </span>
+                  <span class="w-4 text-2xl font-semibold pl-2 self-center">{{
```

Comment:
Nice work making the degrees slightly smaller and less bold than the healthcare professional names! It'll help the names stand out more.

Reply:
Would you suggest a character limit, a degree limit, or some combination of the two. Like probably wouldn't want the UI to cut off in the middle of a degree?